.php_cs.dist

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$finder = new PhpCsFixer\Finder();
+$config = new PhpCsFixer\Config('json-schema', 'json-schema style guide');
+$finder->in(__DIR__);
+
+/* Based on ^2.1 of php-cs-fixer */
+$config
+    ->setRules([
+        // default
+        '@PSR2' => true,
```

Comment:
Sorry for the confusion. I am cool with using php-cs-fixer to evaluate code style and am not tied to phpcs specifically. Adding to the TravisCI flow will be very helpful.

I am fine with the ruleset but am curious if the additional configs override/supplement defaults for the PSR-2 spec or the Symfony spec?